src/map/map.h

```diff
@@ -553,6 +553,7 @@ enum status_point_types { //we better clean up this enum and change it name [Hem
 	SP_MOD_EXP=125,
 	SP_MOD_DROP=126,
 	SP_MOD_DEATH=127,
+	SP_BANKVAULT=128,
```

Comment:
not sure is possible add any custom contants here. I think they related to aegis or client constants?